### PR TITLE
Sets gender on most crewmate plushies (please check PR if you have one)

### PR DESCRIPTION
**Pull request by YakumoChen**
## About The Pull Request
It's very important
@DuffCreeper or whoever else made plushie PRs should probably tell me what else to change for the ones I don't have context on

## Why It's Good For The Game
It is

## Changelog
but not important enough for a changelog

### CODE REVIEW

**Comment by YakumoChen** [comment]
I'm just doing it for posterity on the ones I can identify the correct one for, but I don't think I touched yours anyway yet, so I'll just not.

**Comment by DuffCreeper** [comment]
I mean adding genders isn't completely necessary when plushies are NULL by default and are assigned random genders based on 50/50. If people did mind they can always ask about it being changed

**Comment by YakumoChen** [comment]
I mostly did it for myself but I figured I might as well assign the correct genders to plushies as a default rather than the exception

**Comment by Swiftfeather** [comment]
Sarmie is girl

**Comment by Owai-Seek** [comment]
I had a plushie on oldbase but I don't know if it ever made it over here. 🤷‍♂️

**Comment by SarmentiCampbell** [comment]
> Sarmie is girl

I will strangle you

**Comment by Gandalf2k15** [comment]
Is this ready?

File: modular_skyrat/modules/customization/game/objects/items/plushes.dm [hunk 1]
```diff
@@ -131,6 +134,7 @@
 	attack_verb_continuous = list("baps")
 	attack_verb_simple = list("bap")
 	squeak_override = list('modular_skyrat/modules/emotes/sound/voice/weh.ogg' = 1)
+	gender = FEMALE
```
Comment (SarmentiCampbell) [inline modular_skyrat/modules/customization/game/objects/items/plushes.dm]: wtf

Comment (SarmentiCampbell) [reply modular_skyrat/modules/customization/game/objects/items/plushes.dm]: Sarmie a male bro

Comment (tf-4) [reply modular_skyrat/modules/customization/game/objects/items/plushes.dm]: doubt

Comment (YakumoChen) [reply modular_skyrat/modules/customization/game/objects/items/plushes.dm]: The first person to say so lied, I dont know everyone.

**Comment by YakumoChen** [comment]
@Gandalf2k15 anytime is fine, people can always do it themself later

**Pull request merged**